bin/truffleruby

```diff
@@ -24,6 +24,30 @@ function mtime {
     fi
 }
 
+# Prefix any option in $@ with -J:, values are just copied over
+function prefix_options {
+    local result=()
+    for opt in "$@"
+    do
+        case $opt in
+            -*)
+                result+=("-J:${opt} ") ;;
```

Comment:
Since Bash doesn't allow non-integer return values (duh!), a global variable is probably the next best workaround. `echo` is risking all kind of problems with argument splitting.